tests/test_timeseriesflattener/test_wrapper.py

```diff
@@ -1,8 +1,9 @@
-from utils_for_testing import *
+from pandas.testing import assert_frame_equal
+from timeseriesflattener.create_feature_combinations import create_feature_combinations
 from timeseriesflattener.flattened_dataset import FlattenedDataset
-from timeseriesflattener.resolve_multiple_functions import (
-    get_max_in_group,
-)
+from timeseriesflattener.resolve_multiple_functions import get_max_in_group
+
+from utils_for_testing import *
```

Comment:
Generally considered an antipattern, can lead to multiple errors which are hard to trace. It also reduced readability. I see no reason to use it here.